src/rc_allocator.h

```diff
@@ -44,8 +44,8 @@ _Static_assert(offsetof(rc_allocator_meta_page, splinters) == 0,
  *----------------------------------------------------------------------
  */
 typedef struct rc_allocator_stats {
-   int64 curr_allocated;
-   int64 max_allocated;
+   int64 curr_allocated; // # of extents allocated
+   int64 max_allocated;  // # of extents allocated hwm
```

Comment:
I don't think "hwm" is as widely known as you think it is.  "High-water mark" is fine if you want.  But do you really think "hwm" is clearer than "max"?

Reply:
I think so, yes, Rob.

In this area of metrics and stats-gathering, for things like this one which go up and down, the terms 'HWM' and 'LWM' (low-water mark) are fairly common.

They are usually preferred over 'max' and 'min', which could be interpreted as the "max-value this thing can go to" or the "min-value this thing can drop to"; i.e. hard min/max limits.

I would have changed the name of the variable itself to `hwm_allocated` but refrained as it would be considered unacceptable. But, I think it's better to go with hwm / lwm terminology for such monitoring related metrics variables.